src/System.CodeDom/dir.props

```diff
@@ -1,7 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\dir.props" />
   <PropertyGroup>
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
+    <IsDesktopFacade>true</IsDesktopFacade>
```

Comment:
Actually looking at the targets this will get bin-placed from the src build (the issue title kind of through me off).